src/WpfMath/TexFormulaHelper.cs

```diff
@@ -153,9 +153,34 @@ public void AddOperator(string operatorFormula, string lowerLimitFormula, string
                 formulaParser.Parse(upperLimitFormula), useVerticalLimits);
         }
 
+        public int MoveOffset(Atom atom, int pos)
+        {
+            if (atom.Source != null)
+            {
+                atom.Source = new SourceSpan(atom.Source.Source, pos, atom.Source.Length);
+                int currentPos = pos;
+                foreach (var child in atom.Children)
+                {
+                    currentPos = MoveOffset(child, currentPos);
+                }
+                return pos + atom.Source.Length;
+            }
+            return pos;
+        }
+
         public void AddOperator(string operatorFormula, bool useVerticalLimits)
         {
-            AddOperator(formulaParser.Parse(operatorFormula), null, null, useVerticalLimits);
+            var formula = formulaParser.Parse(operatorFormula);
+            if (formula?.RootAtom.Source != null)
+            {
+                formula.RootAtom.Source = new SourceSpan(source.Source, source.Start, formula.RootAtom.Source.Length);
+                var pos = source.Start + 1;
```

Comment:
Honestly I don't getting why this `+ 1` is here. Please leave a comment about that.